### PR TITLE
postgresql: Remove myself from maintainers

### PR DESCRIPTION
**Pull request by databus23**
I have not been responsive to requests in the past year. In order avoid frustrations  with future PRs I withdraw myself from the list.

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/postgresql/Chart.yaml [hunk 1]
```diff
@@ -12,6 +12,5 @@ icon: https://www.postgresql.org/media/img/about/press/elephant.png
 sources:
 - https://github.com/kubernetes/charts
 - https://github.com/docker-library/postgres
-maintainers:
-- name: databus23
+maintainers: []
```
Comment (davidkarlsen) [inline stable/postgresql/Chart.yaml:15]: chart version needs bump in order to merge and release the chart

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/postgresql/Chart.yaml (hunk 1 above)
Comment (databus23) [reply stable/postgresql/Chart.yaml:15]: done

**Comment by desaintmartin** [comment]
Does it mean this Chart is orphan? How to get existing/new PRs merged?

**Comment by cpanato** [comment]
@desaintmartin it is not orphan there are others maintainers. dont worry

**Pull request merged**